polygon/bridge/db.go

```diff
@@ -52,6 +54,8 @@ type Store interface {
 
 	GetLatestEventID(ctx context.Context) (uint64, error)
 	GetLastProcessedEventID(ctx context.Context) (uint64, error)
+	StoreTxMap(ctx context.Context, txMap map[libcommon.Hash]uint64) error
+	TxMap(ctx context.Context, borTxHash libcommon.Hash) (uint64, bool, error)
 	GetSprintLastEventID(ctx context.Context, lastID uint64, timeLimit time.Time, stateContract abi.ABI) (uint64, error)
 	AddEvents(ctx context.Context, events []*heimdall.EventRecordWithTime, stateContract abi.ABI) error
 	GetEvents(ctx context.Context, start, end uint64) ([][]byte, error)
```

Comment:
This is a note for a follow up PR not for this one but would be nice if we align the naming of the bridge store with the naming of the heimdall store - ie as we decided last week we won't use "GetXYZ" but just "XYZ". Also use "PutEvents" instead of "AddEvents".